tests/tests_tui/test_tui_directorytree.py

```diff
@@ -131,7 +131,10 @@ async def test_create_folders_directorytree_clipboard(
             await self.reload_tree_nodes(
                 pilot, "#create_folders_directorytree", 4
             )
-            pyperclip.copy("STARTING VAL")
+            try:
```

Comment:
Here I think in the test environment, we should never be in headless mode (tests are running okay in the CI). In this case, we can let a `PyperclipException` be raised just so we are aware of any other possible failures.